### PR TITLE
Last Updated Training

### PR DESCRIPTION
**Pull request by two-first-names**
#### Tests
Does this PR include tests for the changes introduced?
- [X] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: src/app/shared/components/trianing-link-panel/trianing-link-panel.component.html [hunk 1]
```diff
@@ -1,25 +1,4 @@
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-one-third">
```
Comment (aaron-russell) [inline src/app/shared/components/trianing-link-panel/trianing-link-panel.component.html:2]: Are these links no longer needed?

Comment (two-first-names) [reply src/app/shared/components/trianing-link-panel/trianing-link-panel.component.html:2]: Yeah, was discussed in meeting after standup this morning.

File: src/app/shared/components/trianing-link-panel/trianing-link-panel.component.component.ts [hunk 2]
```diff
@@ -28,10 +37,29 @@ export class TrainingLinkPanelComponent implements OnInit, OnDestroy {
...
-        if (data && data.id) {
+        if (data && data.id !== undefined) {
           this.establishmentUid = data.uid;
+          this.subscriptions.add(
+            this.workerService.getAllWorkers(data.uid).subscribe(
+              workers => {
+                workers.forEach((worker: Worker) => {
+                  if (worker.trainingCount > 0) {
+                    this.subscriptions.add(
+                      this.workerService.getTrainingRecords(data.uid, worker.uid)
```
Comment (chadtomkiss-madetech) [inline src/app/shared/components/trianing-link-panel/trianing-link-panel.component.component.ts:51]: There's a lot of HTTP requests to get training records for every worker that's shown.

`getAllWorkers` on L46 also loops over each worker on the server side to get all of the training records them.

Not sure if I'm over optimising but do you think amending the server side response to include the latest updated record date would be a more performant option?

File: src/app/shared/components/trianing-link-panel/training-link-panel.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    const router = injector.get(Router) as Router;
+
+    return {
+      component,
+      establishmentService
+    };
+  }
+
+  it('should render a TrainingLinkPanelComponent', async () => {
+    const component = await setup();
```
Comment (chadtomkiss-madetech) [inline src/app/shared/components/trianing-link-panel/training-link-panel.component.spec.ts:57]: This doesn't destructure like in the test on L62

`setup` returns an object with both the component and the establishment service so I'm not sure the naming `component` is technically correct.

Comment (two-first-names) [reply src/app/shared/components/trianing-link-panel/training-link-panel.component.spec.ts:57]: Fair!

File: src/app/shared/components/trianing-link-panel/trianing-link-panel.component.component.ts (hunk 2 above)
Comment (two-first-names) [reply src/app/shared/components/trianing-link-panel/trianing-link-panel.component.component.ts:51]: Worker does have a lastUpdated on it, but no idea if that relates to training or the worker as a whole.

**Pull request merged**